oaknational/Oak-Web-Application
# feat: viewing info for teacher lesson browsing

## Description

- changed isCurrent logic 
- passed down curentStyles prop to highlight which button link had been selected

## Issue(s)

- tierSlug static so wasn't changing 
- using Router to access the current query to compare the slug with the tier to highlight which one is selected 

Fixes #1234 

## How to test

1. Go to {https://deploy-preview-1277--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks4/subjects/maths/units?tier=higher}
2. Click on Core, Higher, Foundation buttons
3. You should see selected button will change colour

## Screenshots

How it used to look (delete if n/a):
<img width="1090" alt="Screenshot 2023-02-16 at 13 55 42" src="https://user-images.githubusercontent.com/91190841/219384251-d006d78f-97f4-4542-8c45-f57d097ea093.png">


How it should now look:

<img width="1187" alt="Screenshot 2023-02-17 at 09 04 46" src="https://user-images.githubusercontent.com/91190841/219601092-99df6755-0983-454d-af72-581599472ca6.png">



## Checklist

- [ x] Added or updated tests where appropriate
- [x ] Manually tested across browsers / devices
- [ x] Considered impact on accessibility
